content/filters/html_filters.md

```diff
@@ -0,0 +1,94 @@
+# HTML Filters
+
+HTML filters wrap assets in HTML tags.
+
+1. TOC
+{:toc}
+
+## asset_url
+
+Add asset file to assets base url path.
+
+#### Input
+
+~~~ liquid
+  {{ 'images/large/' | asset_url }}
+~~~
+
+#### Output
+
+~~~ html
+  "/assets/themes/example_theme/images/large/"
+~~~
+
+## stylesheet_url
+
+Generate url with given stylesheet.
```

Comment:
This comment deny above one. `stylesheet_url` generates url and `stylesheet_tag` suit it up under stylesheet tag with this url.